package.json

```diff
@@ -41,7 +41,7 @@
   "dependencies": {
     "amstock3": "github:amcharts/amstock3",
     "angular": "<=1.6.10",
-    "angular-advanced-searchbox": "github:reyraa/angular-advanced-searchbox",
+    "angular-advanced-searchbox": "github:dnauck/angular-advanced-searchbox",
```

Comment:
I believe we don't use that dependency anymore, can you please remove all references to that?

Reply:
Done.